glossary.md

```diff
@@ -0,0 +1,161 @@
+# Adobe I/O Events Glossary
+
+[A](#a) &bull; [B](#b) &bull; [C](#c) &bull; [D](#d) &bull; [E](#e) &bull; [F](#f) &bull; [G](#g) &bull; [H](#h) &bull; [I](#i) &bull; [J](#j) &bull; [K](#k) &bull; [L](#l) &bull; [M](#m) &bull; [N](#n) &bull; [O](#o) &bull; [P](#p) &bull; [Q](#q) &bull; [R](#r) &bull; [S](#s) &bull; [T](#t) &bull; [U](#u) &bull; [V](#v) &bull; [W](#w) &bull; [X](#x) &bull; [Y](#y) &bull; [Z](#z)
+
+## A
+
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## B
+
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## C
+
+**Consumer:**
+
+**Consumer Org ID:**
+
+[Back to top](#adobe-io-events-glossary)
+
+## D
+
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## E
+
+**Events:**
+
+**Event Code:**
+
+**Event Metadata:**
+
+**Event Provider:** Also known as "Provider"
+
+**Event Types:**
+
+[Back to top](#adobe-io-events-glossary)
+
+## F
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## G
+
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## H
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## I
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## J
+
+**Journalling:**
+
+[Back to top](#adobe-io-events-glossary)
+
+## K
+
+[Back to top](#adobe-io-events-glossary)
+
+## L
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## M
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## N
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## O
+
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## P
+
+**Polling:**
+
+**Project ID:**
+
+**Provider:** Also known as "Event Provider"
+
+**Publisher:**
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## Q
+
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## R
+
+
+[Back to top](#adobe-io-events-glossary)
+
+## S
+
+**Signature verification:**
+
+[Back to top](#adobe-io-events-glossary)
+
+## T
+
+**Tracing:**
```

Comment:
Instead of “Tracing”, we should use “Debug Tracing”. That’s what Console refers externally.